docs/src/VehkamakiPlots.jl

```diff
@@ -0,0 +1,174 @@
+using Plots
+
+include("../../src/Nucleation.jl")
+using .Nucleation
+
+# Reproduces figures from Vehkamaki et al, 2002 doi:10.1029/2002JD002184
+# In order verify that the implementation is correct
+# Nucleation rate shown here is without any adjustment due to self-coagulation
+
+function plot_vehk(
```

Comment:
Do we want to keep this one in plots to compare with our nucleation rates? Or is this not needed?

Reply:
I think it would be good to keep for now in order to compare. The other code doesn't rely on it, so it would be easy to remove later.